forge-gui/res/cardsfolder/upcoming/canyon_vaulter.txt

```diff
@@ -0,0 +1,8 @@
+Name:Canyon Vaulter
+ManaCost:1 W
+Types:Creature Kor Pilot
+PT:3/1
+T:Mode$ Crewed | ValidCrew$ Card.Self | Execute$ TrigPump | TriggerZones$ Battlefield | TriggerDescription$ Whenever this creature saddles a Mount or crews a Vehicle during your main phase, that Mount or Vehicle gains flying until end of turn.
+T:Mode$ Saddled | ValidCrew$ Card.Self | Execute$ TrigPump | TriggerZones$ Battlefield | Secondary$ True | TriggerDescription$ Whenever this creature saddles a Mount or crews a Vehicle during your main phase, that Mount or Vehicle gains flying until end of turn.
+SVar:TrigPump:DB$ Pump | Defined$ TriggeredCard | KW$ Flying
```

Comment:
don't these need *LKICopy for now? :/
or should we try and fix that soon and leave new cards this way?

Reply:
The TriggeredCard there isn't an LKI copy, so i don't think that would cause a problem?
but just in case, i added more LKICopy

Reply:
it's needed currently because without you could flicker it, AbilityUtils would lookup the new object and wrongly affect it